internal/pkg/coordinator/monitor.go

```diff
@@ -192,8 +195,9 @@ func (m *monitorT) handlePolicies(ctx context.Context, hits []es.HitT) error {
 func (m *monitorT) ensureLeadership(ctx context.Context) error {
 	m.log.Debug().Msg("ensuring leadership of policies")
 	err := dl.EnsureServer(ctx, m.bulker, m.version, m.agentMetadata, m.hostMetadata, dl.WithIndexName(m.serversIndex))
+
 	if err != nil {
-		return err
+		return errors.New(fmt.Sprintf("Failed to check server status on Elasticsearch (%s): %s", m.hostMetadata.Name, err.Error()))
```

Comment:
can we wrap existing error with error message rather than using `err.Error()` this way we may be losing some context